frontend/catalyst/compiler.py

```diff
@@ -228,6 +261,8 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
 DEFAULT_PIPELINES = [
     HLO_LOWERING_PASS,
     QUANTUM_COMPILATION_PASS,
+    # note: the position of QUANTUM_CIRCUIT_TRANSFORMS is currently arbitrary
+    QUANTUM_CIRCUIT_TRANSFORMS,
```

Comment:
I think the `QUANTUM_COMPILATION_PASS` pipeline was meant for exactly these kinds of passes, hence the name (people understand quantum compilation as circuit transforms) 😅

Reply:
Oh, did not know that. This indeed would make life easier!